nbbuild/antsrc/org/netbeans/nbbuild/MakeNBM.java

```diff
@@ -1029,6 +1053,29 @@ private Document createInfoXml(final Attributes attr) throws BuildException {
         return doc;
     }
 
+    private Document createFakeOSGiInfo(File osgiJar) {
+        DOMImplementation domimpl;
+        try {
+            domimpl = DocumentBuilderFactory.newInstance().newDocumentBuilder().getDOMImplementation();
+        } catch (ParserConfigurationException x) {
+            throw new BuildException(x, getLocation());
+        }
+
+        log("Creating fake info.xml for OSGi bundle", Project.MSG_VERBOSE);
+        
+        String pub = "-//NetBeans//DTD Autoupdate Module Info 2.5//EN";
+        String sys = "http://www.netbeans.org/dtds/autoupdate-info-2_5.dtd";
+        Document doc = domimpl.createDocument(null, "module", domimpl.createDocumentType("module", pub, sys));
+        
+        try (JarFile jf = new JarFile(osgiJar)) {
+            MakeUpdateDesc.fakeOSGiInfoXml(jf, osgiJar, doc);
```

Comment:
I noticed a difference between regular modules and OSGI(fake) modules: In the normal modules a license section is present in the `info.xml`. The LICENSE is present in the generated `nbm`. So from that perspective this should be good - I'm just wondering whether we need to cover this.